contrastsecurity/README.md

```diff
@@ -0,0 +1,44 @@
+## Contrast Security - Protect ScreenBoard for DataDog
+
+### Set up Contrast Protect logs collection
+
+Enable logs collection for Datadog Agent in `/etc/datadog-agent/datadog.yaml` on Linux platforms. On other platforms, refer to the [Agent Configuration Files guide](https://docs.datadoghq.com/agent/guide/agent-configuration-files/?tab=agentv6) for the location of your configuration file:
+```
+logs_enabled: true
+```
+
+* Create a new folder `java.d` in the `conf.d/` directory of DataDog configuration directory.
```

Comment:
```suggestion
* Add this configuration block to your `contrastsecurity.d/conf.yaml` file to start collecting your Contrast Logs:
```

Reply:
is this change mandatory? Its just how our docs present it